typedoc.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
I think it would be easier to maintain this config if it was in JS: https://typedoc.org/documents/Options.Configuration.html#javascript-files

One benefit would be to automatically generate a list of entry points based on current state of `packages/*` folder